src/main/resources/application.yml

```diff
@@ -6,4 +6,4 @@ management:
       exposure:
         include: refresh, configprops
 application:
-  secret: "sm://demo-secret"
\ No newline at end of file
+  secret: ${sm://demo-secret}
```

Comment:
In [this doc](https://googlecloudplatform.github.io/spring-cloud-gcp/reference/html/index.html#secret-manager-property-source), you can't put `${}` around the secret id